frontend/src/App.tsx

```diff
@@ -33,9 +34,6 @@ import store from './redux/stores/store';
 
 const useStyle = makeStyles(theme => ({
   root: {
-    display: 'flex',
```

Comment:
Would this patch be simpler if we were using Redux to set/retrieve the theme name? Themes like this patch changes a lot of code just for the theme name to be changed by an inner component.

Reply:
No, So here the problem is not that we aren't getting the correct theme, we are but we are using the useStyles for App before we could even get the new theme via ThemeProvider that's why this patch renders the entire app in a container and that container is wrapped by ThemeProvider and in that container itself we use the useStyles and not outside the ThemeProvider. I hope i didn't confused you :sweat_smile:

Reply:
@joaquimrocha i would prefer to use the settings redux state to store it locally to be done in a different pr